src/Sulu/Bundle/AdminBundle/Resources/js/containers/DeleteReferencedResourceDialog/DeleteReferencedResourceDialog.js

```diff
@@ -0,0 +1,78 @@
+// @flow
+import React from 'react';
+import Dialog from '../../components/Dialog';
+import {translate} from '../../utils';
+import type {Resource} from '../../types';
+
+type Props = {
+    allowDeletion: boolean,
+    loading: boolean,
+    onCancel: () => void,
+    onConfirm: () => void,
+    referencingResources: Resource[],
+    referencingResourcesCount: number,
+}
```

Comment:
do we need to add a `open` prop that is passed to the `Dialog` to keep the animation?